### DISCUSSION TITLE
How do I put more than one line in a line chart? C# docs lacking, API unclear

### DISCUSSION THREAD

**Discussion by StenPetrov** [plotly/Plotly.NET#451] [Q&A]
I want to put multiple lines on the same line chart, then export the chart to PNG.
It's clear the C# API was slapped on as an afterthought, so I'm having trouble figuring this out.
Is it doable? How?

**Comment by kMutagene** [comment]
You are looking for `Chart.Combine`.

I could ask if you are using the native C# API or the base lib and provide code snippets for both cases, but i choose not to due to your passive agressive question.

**Comment by StenPetrov** [comment]
I’ve moved on to ScottPlot, it’s API is C# native, so for me it’s easier to find things/Sten ✉️On Apr 17, 2024, at 8:48 AM, Kevin Schneider ***@***.***> wrote:﻿
You are looking for Chart.Combine.
I could ask if you are using the native C# API or the base lib and provide code snippets for both cases, but i choose not to due to your passive agressive question.

**Reply by kMutagene** [reply]
Good for you
